sources/packages/backend/apps/api/src/services/application-offering-change-request/application-offering-change-request.service.ts

```diff
@@ -344,6 +348,30 @@ export class ApplicationOfferingChangeRequestService {
     return applicationOfferingChangeRequest;
   }
 
+  /**
+   * Get the Application Offering Change Request Status by id.
```

Comment:
Not a part of the PR. Line 335 of the same file is missing select. I believe it is from last story. I would recommend either to handle in upcoming PR or atleast have a todo comment for it. 

![image](https://github.com/bcgov/SIMS/assets/54600590/1c5012c3-1b1b-491a-96ac-f72b3bb13787)

Reply:
Fixed it 👍 . Thanks @dheepak-aot

Reply:
thanks for doing the change.